config/webpack.config.js

```diff
@@ -158,7 +164,8 @@ module.exports = function (env, argv) {
             alias: {
                 // Use vue with the runtime compiler (needed for template strings)
                 // To-do: Remove use of template strings, https://github.com/FlowFuse/flowfuse/issues/3290
-                vue: 'vue/dist/vue.esm-bundler.js'
+                vue: 'vue/dist/vue.esm-bundler.js',
+                '@': path.resolve('frontend/src/ui-components')
```

Comment:
think we may want to go to `frontend/src/` for this, then add the `ui-components` into `line 88`. There is a little legacy here with `ui-components` that we need to clean at some point. The TLDR is that it used to be a standalone repository, then it got silly to maintain, so we merged it into `ui-components`. The `@` alias I've traditionally used for `/src`, and that feels cleaner to me.

Reply:
SGTM! I wanted to ask about the ui-components dir, also about the stylesheets directory. I've regularly grouped them into an assets dir, which included images fonts among other things. We can definitely sort them out

Reply:
Yeah, it's on that list of "nice to haves" that hasn't ever seen the light of day due to other, more urgent, priorities